.env.sample

```diff
@@ -0,0 +1,4 @@
+API_BASE_URL="https://survey-api-staging.nimblehq.co"
```

Comment:
Why not using the production environment instead? Staging can break if we make changes.